src/components/button.component.js

```diff
@@ -0,0 +1,218 @@
+import React, { Component } from 'react';
+import { Platform } from 'react-native';
+import { Button as BaseButton } from 'react-native-elements';
+import { colors, fonts, normalize } from 'config';
+
+const types = {
+  default: {
+    buttonStyle: {
+      borderColor: '#adadad',
```

Comment:
What do you guys think about having all this hex values into another file so we can reuse them later on?

Reply:
Definitely. Still waiting on Houssein to decide what colors/types we should define and use UX wise ;)

Reply:
I know we haven't properly dealt with all the colors in the app in the best way and that's my fault :O. Literally `colors` and `languageColors` objects in `colors.js`.

Ideally I would like this and all the colors to live in a `colors` directory or `colors.js` file if it can all fit there nicely. Are all the colors here custom? I'm okay with merging this in now and having that discussion to refactor all the colors in our app after our release if you like. Otherwise this PR looks _solid_